lib/WebpackOptionsApply.js

```diff
@@ -588,7 +588,7 @@ class WebpackOptionsApply extends OptionsApply {
 					break;
 				}
 				default:
-					// @ts-ignore never is expected here
+					// TODO options.cache.type type is : "memory" | "filesystem"
```

Comment:
@sokra Are you expecting `type` to be `never` here?

Reply:
yes, memory and filesystem is handled in the switch cases above.

Reply:
Ohhh it took me a while to figure this out. So `cacheOptions.type` is indeed `never`. Here we're using `options.cache.type` which its type is not being narrowed with the switch statement. TypeScript id working as expected. I'll remove the comment.